.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
@prestonbourne can we remove the top-level configs? Linting should be handled at the workspace level and coordinated with turbo